cli/root.go

```diff
@@ -91,7 +92,7 @@ func init() {
 	}
 	sort.Strings(backends)
 	RootCmd.PersistentFlags().StringVarP(&manager.ServiceManager, "svcmgr", "m", "", fmt.Sprintf("service manager, must be one of: %s", strings.Join(backends, ", ")))
-	RootCmd.PersistentFlags().StringVarP(&manager.Before, "before", "t", "", "how long before certificates expire to start renewing (in the form Nh)")
+	RootCmd.PersistentFlags().DurationVarP(&manager.Before, "before", "t", time.Hour * 72, "how long before certificates expire to start renewing (in duration format)")
```

Comment:
Can you just use `mgr.DefaultBefore` as the default value here?

Reply:
Could and should, will follow up.